sufia-models/app/services/sufia/generic_file_indexing_service.rb

```diff
@@ -5,6 +5,7 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('label')] = object.label
         solr_doc[Solrizer.solr_name('file_format')] = object.file_format
         solr_doc[Solrizer.solr_name('file_format', :facetable)] = object.file_format
+        solr_doc[Solrizer.solr_name(:file_size, :symbol)] = object.file_size[0]
```

Comment:
@mjgiarlo I do not know.  The file_size field was defined previously to this commit.  I am just using the existing field.